website/src/app/[lang]/[country]/(website)/books/section-1.tsx

```diff
@@ -0,0 +1,29 @@
+import { DefaultPageProps } from '@/app/[lang]/[country]';
+import { BaseContainer, Typography } from '@socialincome/ui';
+
+export default async function Section1({ params }: DefaultPageProps) {
+	return (
+		<BaseContainer className="mt-20 flex flex-col content-center justify-center text-center">
+			<Typography as="div" size="4xl" weight="bold" lineHeight="loose" className="m-3 w-auto pb-4">
+				Recommended Reading
```

Comment:
Can you put these translations also into the `website-books.json` file and use the Translator as in section-2.tsx?